doc/source/api.rst

```diff
@@ -111,9 +111,7 @@ Google BigQuery
    read_gbq
    to_gbq
    generate_bq_schema
```

Comment:
I think we should rename this as well. Can you deprecate ``generate_bq_schema``, with a rename to ``generate_schema``. (rename the function and provide the original as a wrapper witth the warning, then call the renamed)